core/flyout.js

```diff
@@ -1388,3 +1229,12 @@ Blockly.Flyout.prototype.reflow = function() {
     this.workspace_.addChangeListener(this.reflowWrapper_);
   }
 };
+
+/**
+ * @return {boolean} True if this flyout may be scrolled with a scrollbar or by
+ *     dragging.
+ * @package
+ */
+Blockly.Flyout.prototype.isScrollable = function() {
```

Comment:
how is this used?  Also, ooc, when do we have a scrollbar that isn't visible? Just when blockly is set to be hidden?